lib/OpenQA/Schema/ResultSet/Jobs.pm

```diff
@@ -112,6 +112,13 @@ sub create_from_settings {
     my ($self, $settings, $scheduled_product_id) = @_;
 
     my %settings = %$settings;
+    foreach my $key (keys %settings) {
+        my $value = $settings{$key};
+
+        if (ref $value eq 'ARRAY' || ref $value eq 'HASH') {
+            $settings{$key} = encode_json($value);
```

Comment:
Does this work with non-ASCII characters like umlauts? If not you might need to encode this to UTF-8.

Reply:
you're right, they are not printed correctly, i'll do that :+1: